openpype/hosts/resolve/api/plugin.py

```diff
@@ -402,6 +402,9 @@ def load(self):
         if handle_end is None:
             handle_end = int(self.data["assetData"]["handleEnd"])
 
+        self.timeline_in = int(self.data["assetData"]["clipIn"])
```

Comment:
Interesting - I though the PR description meant it would place the loaded item at the current playbar time (like at current position) and not use the clip in from the asset data.

Also, there's no reason this is stored on `self` - please just make it `timeline_in =` etc. and use it as the var like that. Right?